test/parallel_api/experimental/asynch.pass.cpp

```diff
@@ -117,6 +117,10 @@ void test2_with_buffers()
 void test_with_usm()
 {
     cl::sycl::queue q;
+
+    if(!TestUtils::has_aspect(q.get_device(), sycl::aspect::usm_shared_allocations))
+        return;
```

Comment:
I think that this function should should become boolean. And when we return from this function, we should return as in skip test vase from main.

Reply:
The buffers will be tested even If ```test_with_usm``` is skipped. Given that modifying return seems unnecessary.
Do you think there should be a notification that the test case with USM has been skipped (e.g. via some message in the output)?